include/tvm/relay/attrs/vision.h

```diff
@@ -12,6 +12,34 @@
 namespace tvm {
 namespace relay {
 
+/*! \brief Attributes used in multibox_prior operators */
+struct MultiBoxPriorAttr : public tvm::AttrsNode<MultiBoxPriorAttr> {
```

Comment:
I think for consistency with other files, you should probably name it `MultiBoxPriorAttrs` (with the final "s")